app/build.gradle

```diff
@@ -370,6 +370,8 @@ android.applicationVariants.all { variant ->
             versionCode = versionCode + (2 * multiplier)
         } else if (variant.flavorName.contains("Arm")) {
             versionCode = versionCode + (1 * multiplier)
+        } else if (variant.flavorName.contains("Universal")) {
```

Comment:
Universal variants don't exist anymore in the latest code.

Reply:
@colintheshots This is landing in 6.1.1 🙌